### PR TITLE
Only search for queries created by the pollbot current user.

### LINKED ISSUE #195: Let's only search for the STMO user when looking at existing telemetry request
This would be useful to make sure that we can have a request for Stage and a request for Prod without 403 when one tries to edit the other user requests.

### PR DESCRIPTION
**Pull request by Natim**
Fixes #195

### CODE REVIEW

File: tests/test_tasks.py [hunk 1]
```diff
@@ -1388,6 +1403,34 @@ def _telemetry_mock_query_result(self, body):
             "Telemetry uptake calculation for version 57.0a1 ({}) is in progress".format(build_id)
         )
 
+    async def test_telemetry_update_uptake_creates_the_query_if_not_found_for_this_user(self):
+        build_id = "{}192146".format(yesterday(formating='%Y%m%d'))
```
Comment (magopian) [inline tests/test_tasks.py]: why not generate also the time, on top of the date? `192146` is 19:21:46? Or am I missing something?

File: tests/test_app.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+from pollbot.app import get_app
+
+
+def test_get_app_fails_if_telemetry_user_id_is_not_defined():
+    initial_telemetry_user_value = os.getenv("TELEMETRY_USER_ID")
+    try:
+        os.environ["TELEMETRY_USER_ID"] = ""
+        get_app()
+    except SystemExit as e:
+        assert e.code == 1
```
[the remark this answers is not in the document]
Comment (Natim) [reply tests/test_app.py]: At first that what I wanted to do but if we get the SystemExit with this error code it is really likely that the print was successful.

File: Makefile [hunk 3]
```diff
@@ -57,7 +57,7 @@ serve: install version-file
 	$(VENV)/bin/pollbot
 
 tests-once: install-dev version-file
-	$(VENV)/bin/py.test --cov-report term-missing --cov-fail-under 100 --cov pollbot tests -s
+	TELEMETRY_USER_ID=502 $(VENV)/bin/py.test --cov-report term-missing --cov-fail-under 100 --cov pollbot tests -s
```
[the remark this answers is not in the document]
Comment (Natim) [reply Makefile]: Do you have an idea of how to do it?

File: tests/test_tasks.py [hunk 4]
```diff
@@ -1218,15 +1218,21 @@ def _telemetry_mock_release_query(self, body=None, *, build_id='20171009192146')
             body = [{
                 "latest_query_data_id": 5678,
                 "id": 40197,
-                "name": "Uptake Firefox RELEASE 57.0 (20171009192146)"
+                "name": "Uptake Firefox RELEASE 57.0 (20171009192146)",
+                "user": {
+                    "id": os.getenv("TELEMETRY_USER_ID")
```
[the remark this answers is not in the document]
Comment (Natim) [reply tests/test_tasks.py]: Yes :+1:

File: tests/test_tasks.py (hunk 1 above)
Comment (Natim) [reply tests/test_tasks.py]: I guess yesterday is using a date object so we don't have access to the time, however it is just a way to build a build ID and the current time is not related to the build ID time.

File: tests/test_app.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (magopian) [reply tests/test_app.py]: sure, but I was wondering about the test coverage. No big deal

File: Makefile (hunk 3 above)
[the remark this answers is not in the document]
Comment (magopian) [reply Makefile]: i think there are some libraries that do that (iirc something called "env"? I can't find it anymore, it's difficult to google), base on files in folders, and this utility will automatically load those env vars when launched... not sure that would do the trick though (and it adds one library to the stack)

**Pull request merged**